packages/gensx/src/commands/new.ts

```diff
@@ -393,7 +393,7 @@ To get started:
   ${projectPath !== "." ? pc.cyan(`cd ${projectPath}`) : ""}
   ${pc.cyan(template.runCommand)}
```

Comment:
Here we currently print:

```
To get started:
  cd gensxsandbox
  OPENAI_API_KEY=<your_api_key> npm run dev
```
I didn't update the run command so it still shows how to run locally first but curious if anyone thinks we should update this.

Reply:
Well the getting started in the readme is "deploy first", we should probably be consistent.

Reply:
fair. Maybe it should show all the major commands. Something like:

```
To get started:
  cd gensxsandbox
  
From there, you can:
- Deploy the project: OPENAI_API_KEY=<your_api_key> npm run deploy
- Run the project locally: OPENAI_API_KEY=<your_api_key> npm run dev
- Run the API server locally: OPENAI_API_KEY=<your_api_key> npm start
```

Reply:
can probably figure out slightly better formatting